modules/dmrpp_module/DmrppArray.h

```diff
@@ -119,16 +144,41 @@ class DmrppArray : public libdap::Array, public dmrpp::DmrppCommon {
             const vector<unsigned long long> &constrained_array_shape);
 
 public:
-    DmrppArray(const std::string &n, libdap::BaseType *v) : libdap::Array(n, v, true /*is dap4*/), DmrppCommon() { }
+    DmrppArray(const std::string &n, libdap::BaseType *v) :
+            libdap::Array(n, v, true /*is dap4*/),
+            DmrppCommon(),
+            is_variable_length_string_array(false),
```

Comment:
It's probably better to initialize these where they are defined in the class and not here in the constructors since that will eliminate duplication and provide the proverbial single source of truth. I bet these are some of the 21 'code smells' that sonar is reporting.